<bug>
          <bug_id>4748</bug_id>
          
          <creation_ts>2011-03-08 09:13:00 +0100</creation_ts>
          <short_desc>Tests in qx.test.ui.core.Command fail</short_desc>
          <delta_ts>2011-03-16 09:33:38 +0100</delta_ts>
          <reporter_accessible>1</reporter_accessible>
          <cclist_accessible>1</cclist_accessible>
          <classification_id>1</classification_id>
          <classification>Unclassified</classification>
          <product>framework</product>
          <component>app.testrunner</component>
          <version>1.3</version>
          <rep_platform>All</rep_platform>
          <op_sys>All</op_sys>
          <bug_status>CLOSED</bug_status>
          <resolution>FIXED</resolution>
          
          
          
          
          
          <priority>P2</priority>
          <bug_severity>normal</bug_severity>
          <target_milestone>1.3.1</target_milestone>
          
          
          
          <everconfirmed>1</everconfirmed>
          <reporter name="Daniel Wagner">daniel.wagner</reporter>
          <assigned_to name="Martin Wittemann">martin.wittemann</assigned_to>
          
          <qa_contact name="Daniel Wagner">daniel.wagner</qa_contact>
          <cf_browser>---</cf_browser>
          

      

      

      

          <long_desc isprivate="0">
            <commentid>23136</commentid>
            <who name="Daniel Wagner">daniel.wagner</who>
            <bug_when>2011-03-08 09:13:25 +0100</bug_when>
            <thetext>All tests in the qx.test.ui.core.Command class currently fail in the 1_3_x branch with the following error message:

Error tearing down test: Error: Binding could not be found!

What&#39;s worse, they also cause any following test to fail with the same message and confuse the Testrunner: Two results are displayed for each test, messing up the queue count.</thetext>
          </long_desc>
          <long_desc isprivate="0">
            <commentid>23159</commentid>
            <who name="Martin Wittemann">martin.wittemann</who>
            <bug_when>2011-03-09 08:40:10 +0100</bug_when>
            <thetext>I&#39;ll take care of that. This is only 1.3.x.</thetext>
          </long_desc>
          <long_desc isprivate="0">
            <commentid>23256</commentid>
            <who name="Martin Wittemann">martin.wittemann</who>
            <bug_when>2011-03-14 16:22:35 +0100</bug_when>
            <thetext>Fixed that for 1.3.x. with r25639</thetext>
          </long_desc>
      
      

    </bug>